pyZcash/rpc/ZDaemon.py

```diff
@@ -42,13 +42,31 @@ def getBlockHash(self, blockheight):
 	def getBlockByHash(self, blockhash):
 		return self._call('getblock', blockhash)
 
+	def getrawtransaction(self, txid):
+		return self._call('getrawtransaction', txid, 1)
+
 	def getBlockByHeight(self, blockheight):
```

Comment:
The methods added here are named using an all-lowercase convention, but the methods around them are camelCase. Could they be updated to follow the convention of the methods around them?